src/daft-plan/src/logical_plan.rs

```diff
@@ -210,16 +209,7 @@ impl LogicalPlan {
             Self::Project(projection) => projection.multiline_display(),
             Self::Filter(Filter { predicate, .. }) => vec![format!("Filter: {predicate}")],
             Self::Limit(Limit { limit, .. }) => vec![format!("Limit: {limit}")],
-            Self::Explode(Explode { to_explode, .. }) => {
-                vec![format!(
-                    "Explode: {}",
-                    to_explode
-                        .iter()
-                        .map(|e| e.to_string())
-                        .collect::<Vec<_>>()
-                        .join(", ")
-                )]
-            }
+            Self::Explode(explode) => explode.multiline_display(),
```

Comment:
could be cleaner to require all inner structs implement `multiline_display` and do
```
        match self {
            Self::Source(inner) |  Self::Project(inner) | ... => inner.multiline_display(),

```

Reply:
Unfortunately, such an `inner` shared binding in a single match arm means that `inner` must have the same type  for all alternatives.